test/parallel/test-startup-empty-regexp-statics.mjs

```diff
@@ -0,0 +1,26 @@
+// We must load the CJS version here because the ESM wrapper call `hasIPv6`
```

Comment:
The ESM `common/index.mjs`, you mean? Could it perhaps reset `RegExp` after it compiles `hasIPv6`, or is that something we could refactor to not use regex?

Reply:
Not sure it's worth it, loading the CJS file seems to be enough – IMO it's unlikely that any other test would be impacted by that.